test/cpp/api/util.h

```diff
@@ -0,0 +1,45 @@
+#include <torch/nn/module.h>
+#include <torch/nn/modules/linear.h>
+
+#include <memory>
+#include <stdexcept>
+#include <string>
+#include <utility>
+
+namespace torch {
+
+// Lets you use a container without making a new class,
+// for experimental implementations
+class SimpleContainer : public nn::CloneableModule<SimpleContainer> {
+ public:
+  virtual variable_list forward(variable_list) {
+    throw std::runtime_error(
+        "SimpleContainer has no forward, maybe you"
+        " wanted to subclass and override this function?");
+  }
+
+  template <typename Derived>
+  std::shared_ptr<Derived> add(
+      std::shared_ptr<Derived> module,
+      std::string name = std::string()) {
+    Module::add(module, std::move(name));
+    return module;
+  }
+};
+
+struct SigmoidLinear : nn::Module {
```

Comment:
It's in `util.h` because it's used in two different test files (notice this is `test/...`).
They need to be placed in a `Sequential`, and I haven't created activation modules yet to compose them like in PyTorch

Reply:
Ah I didn't see this, but I think you should use the Functional module to define these simple activation functions. I wanted to keep the nn:: namespace as clean a possible.